Assets/Scripts/BrushCatalog.cs

```diff
@@ -181,6 +181,34 @@ public void BeginReload()
             }
         }
 
+
+        public Brush[] GetTagFilteredBrushList()
+        {
+            string[] includeTags = App.UserConfig.Brushes.IncludeTags;
+            string[] excludeTags = App.UserConfig.Brushes.ExcludeTags;
+
+            if (includeTags == null)
+            {
+                Debug.LogError("There will be no brush because none of them are tagged 'include'");
+            }
+
+            // Filter m_GuiBrushList down to those that are both 'eligible' and not 'excluded'
```

Comment:
please replace `'eligible'` with `'included'` to align with the newer name

Reply:
✔